core/passport/github.js

```diff
@@ -2,54 +2,96 @@
  * core/passport/github.js
  * Setup Github Passport object
  *
- * @exports {Function} Server - Sets up Koa server with Github Passport
- * @exports {Object} Route - Koa router object for authentication
+ * @exports {Class} Strategy - Passport class for GitHub login
+ * @exports {Object} Route - Koa router object for GitHub authentication
  */
 
 import Passport from 'koa-passport'
 import Github from 'passport-github'
 import Router from 'koa-router'
 
-import { Config } from '~/app'
+import { Config, Log, Request } from '~/app'
 import { User } from '~/core/model/user'
 
+/**
+ * Updates user with rights given by GitHub
+ *
+ * @param {Object} user - user database object
+ * @return {Object} updated user object
+ */
+const getRights = async function (user) {
+  const promiseMember = function (member, username) {
```

Comment:
maybe rename this function, the function name is not really descriptive of what it does. Also maybe move it outside of the scope of another function.